parsers/DK.py

```diff
@@ -43,13 +40,18 @@ def fetch_production(zone_key='DK-DK1', session=None,target_datetime=None,
                      
     
     url = 'https://api.energidataservice.dk/datastore_search_sql?sql={}'.format(sqlstr)
-    
     response = r.get(url)
-    
-    assert response.status_code == 200 and response.json()['result']['records'] != [], \
-        'Exception when fetching production for ' \
-        '{}: error when calling url={}'.format(zone_key, url)
-        
+
+    if response.status_code != 200:
+        error = response.json()['error']['__type']
+        text = response.json()['error']['info']['orig']
+        logger.warning('"{}" fetching production data for {}: {}'.format(
```

Comment:
here I would actually `raise Error(...)`

Reply:
Sure. `requests.exception.HTTPError` or `requests.exception.ConnectionError` would do?

Reply:
right! You pick :)